app/models/user.rb

```diff
@@ -2,7 +2,9 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :posts, dependent: :destroy
   has_many :likes, dependent: :nullify
-  validates :name, presence: true
+
+  validates :Name, presence: true
+  validates :PostsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/173168856-9b49bdae-e6e5-45a3-b57b-8535d5cb2243.png)

![image](https://user-images.githubusercontent.com/65068771/173168892-4b2c56c4-a9b8-4726-bce5-5dcad0b33b79.png)

- [ ] Kindly the errors above are appearing because you capitalized the name of columns in your validation. Kindly use `name` and `postsCounter` instead of `Name`. and `PostsCounter`

Reply:
Thanks! :100: